sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -106,7 +106,7 @@ private[spark] object HiveUtils extends Logging {
     .doc("When set to true, the built-in ORC reader and writer are used to process " +
       "ORC tables created by using the HiveQL syntax, instead of Hive serde.")
     .booleanConf
-    .createWithDefault(false)
+    .createWithDefault(true)
```

Comment:
This change was made in https://issues.apache.org/jira/browse/SPARK-15705. 

This issue has been resolved?

Reply:
Yes, it's resolved as you see the last my comment on the JIRA.

Reply:
On the JIRA, there is a result on 2.1.1 and 2.2.0. And the following is the result on 2.2.1.
```scala
scala> sql("set spark.sql.hive.convertMetastoreOrc=true")
scala> spark.table("default.test").printSchema
root
 |-- id: long (nullable = true)
 |-- name: string (nullable = true)
 |-- state: string (nullable = true)

scala> spark.version
res2: String = 2.2.1
```

Reply:
By this PR: https://github.com/apache/spark/pull/19470 ?

Reply:
I think it's before #19470 because it's fixed on 2.1.1.

Reply:
How about https://issues.apache.org/jira/browse/SPARK-15757?

Reply:
Yep. That is resolved via https://issues.apache.org/jira/browse/SPARK-15757 by me.

Reply:
Ur, please wait a moment. I'll double check the case to make it sure.

Reply:
Yep. It's resolved via SPARK-14387. The following is a result from 2.2.1.
```sql
hive> CREATE TABLE source(inv_date_sk INT, inv_item_sk INT, inv_warehouse_sk INT, inv_quantity_on_hand INT);
hive> INSERT INTO source VALUES(1,1,1,1);
hive> CREATE TABLE inventory(inv_date_sk INT, inv_item_sk INT, inv_warehouse_sk INT, inv_quantity_on_hand INT)
    > ROW FORMAT DELIMITED FIELDS TERMINATED BY '|' STORED AS ORC;
hive> INSERT OVERWRITE TABLE inventory SELECT * FROM source;

scala> sql("set spark.sql.hive.convertMetastoreOrc=true")
scala> sql("SELECT * FROM inventory").show
+-----------+-----------+----------------+--------------------+
|inv_date_sk|inv_item_sk|inv_warehouse_sk|inv_quantity_on_hand|
+-----------+-----------+----------------+--------------------+
|          1|          1|               1|                   1|
+-----------+-----------+----------------+--------------------+
scala> spark.version
res2: String = 2.2.1
```